client/src/components/ActualTask/ActualTask.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+import './ActualTask.scss';
+
+const ActualTask = ({ actualTask: { description }, status, onChangeStatus }) => {
+  const [connectionStatus, setConnectionStatus] = useState(true);
+
+  // Vibration
+  navigator.vibrate = navigator.vibrate || navigator.webkitVibrate || navigator.mozVibrate || navigator.msVibrate;
+
+  const vibrateStart = ms => {
+    navigator.vibrate(ms);
+  };
+
+  // User Agent
+  var browserArray = ['Chrome', 'Firefox', 'Safari', 'Opera', 'MSIE', 'Trident', 'Edge'];
+  var browser,
+    userAgent = navigator.userAgent;
+
+  for (var i = 0; i < browserArray.length; i++) {
+    if (userAgent.indexOf(browserArray[i]) > -1) {
+      browser = browserArray[i];
+      break;
+    } else {
+      browser = 'Unknown device';
+    }
+  }
+
+  if (browser === 'MSIE' || browser === 'Trident' || browser === 'Edge') {
+    browser = 'Internet Exporer';
+  }
+
+  // Online & Offline
+
+  useEffect(() => {
+    window.addEventListener('online', connectionHandler);
+    window.addEventListener('offline', connectionHandler);
+
+    return () => {
+      window.removeEventListener('online', connectionHandler);
+      window.removeEventListener('offline', connectionHandler);
+    };
+  });
```

Comment:
`}, []);`
If you add empty array, event listeners are added only when component is mounted. If you missed it, event listeners are added during each component update.